src/mesido/workflows/multicommodity_simulator_workflow.py

```diff
@@ -515,7 +515,8 @@ def energy_system_options(self):
 
         self.gas_network_settings["head_loss_option"] = HeadLossOption.LINEARIZED_N_LINES_EQUALITY
         self.gas_network_settings["network_type"] = NetworkSettings.NETWORK_TYPE_HYDROGEN
-        self.gas_network_settings["minimize_head_losses"] = True
+        self.gas_network_settings["minimize_head_losses"] = False
+        self.gas_network_settings["maximum_velocity"] = 60.0
```

Comment:
This looks like a massive maximum_velocity? Is this on purpose?

Reply:
For hydrogen this is the set maximum velocity of 60 m/s.